src/tests/class_tests/openms/source/PepXMLFile_test.cpp

```diff
@@ -62,32 +62,24 @@ START_SECTION(~PepXMLFile())
 delete ptr;
 END_SECTION
 
-START_SECTION(void load(const String& filename, std::vector<ProteinIdentification>& proteins, std::vector<PeptideIdentification>& peptides, const String& experiment_name, const MSExperiment<>& experiment, bool use_precursor_data = false))
+START_SECTION(void load(const String& filename, std::vector<ProteinIdentification>& proteins, std::vector<PeptideIdentification>& peptides, const String& experiment_name, SpectrumMetaDataLookup& lookup))
 {
-  vector<ProteinIdentification> proteins, proteins2;
-  vector<PeptideIdentification> peptides, peptides2;
+  vector<ProteinIdentification> proteins;
+  vector<PeptideIdentification> peptides;
   String pep_file = OPENMS_GET_TEST_DATA_PATH("PepXMLFile_test.pepxml");
   String mz_file = OPENMS_GET_TEST_DATA_PATH("PepXMLFile_test.mzML");
   String exp_name = "PepXMLFile_test";
   MSExperiment<> experiment;
   MzMLFile().load(mz_file, experiment);
-  // load with precursor information:
-  file.load(pep_file, proteins, peptides, exp_name, experiment, true);
+  SpectrumMetaDataLookup lookup;
+  lookup.readSpectra(experiment.getSpectra());
+  file.load(pep_file, proteins, peptides, exp_name, lookup);
+  TEST_EQUAL(peptides.size(), 18);
+  TEST_EQUAL(proteins.size(), 2);
   PeptideIdentification first = peptides[0];
-  TEST_EQUAL(first.getRT(), 0.5927);
-  TEST_EQUAL(first.getMZ(), 538.605);
-  // load without precursor information (tested more thoroughly below):
-  file.load(pep_file, proteins2, peptides2);
-  TEST_EQUAL(peptides.size(), peptides2.size());
-  TEST_EQUAL(proteins.size(), proteins2.size());
-  // check that only m/z and RT differ between "load" methods:
-  for (Size i = 0; i < peptides.size(); ++i)
-  {
-    peptides[i].setRT(peptides2[i].getRT());
-    peptides[i].setMZ(peptides2[i].getMZ());
-  }
-  TEST_EQUAL(peptides == peptides2, true);
-  TEST_EQUAL(proteins == proteins2, true);
+  TEST_REAL_SIMILAR(first.getRT(), 1.3653);
```

Comment:
Previously it used the precursor RTs instead of the MS2 RTs, which is an option that I removed. (I could actually put it back in now that precursor RTs can be picked up as spectrum meta data, but I don't see the need for it.)